### DISCUSSION TITLE
i know this might be a bit stupid, but how do i set a photon appid?

### DISCUSSION THREAD

**Discussion by fluffykanade** [RealMCoded/Rec.js#6] [Q&A] [answered]
¯\_(ツ)_/¯

**Comment by RealMCoded** [comment, marked as answer]
Hello there! I would refer you to my [build modification guide](https://github.com/RealMCoded/Rec.js/wiki/Build-Modification-Guide), but since that is unfinished because i am lazy, i can just do a quick write up here. This expects that you already have a PUN and Voice server app ID.

Firstly, you need to get a hex editor. I use [HXD](https://mh-nexus.de/en/hxd/), but any other one should work.

Open the file `resources,assets` (located in `Recroom_Release_Data`) in HxD, it should look something like this:
![image](https://github.com/RealMCoded/Rec.js/assets/34777184/67a3e5cb-ba4d-4ae8-8fd2-c0d0beedcabd)

Press CTRL+F, type in "photon" into the box that appears
![image](https://github.com/RealMCoded/Rec.js/assets/34777184/5835ef73-e106-422d-b979-08e7534e7b5b)

You should find something that looks like this:
![image](https://github.com/RealMCoded/Rec.js/assets/34777184/be3dbe6e-6ab0-419f-9a33-1c006d67a407)

From there you just need to replace the numbers with the appropriate IDs, The first ID is your PUN, the second is your Voice.

**Reply by fluffykanade** [reply]
thanks :)

**Comment by a-Beanz** [comment]
Thats rather helpful thanks man
